calico-cloud_versioned_docs/version-3.16/image-assurance/understanding-scan-results.mdx

```diff
@@ -118,8 +104,33 @@ Images with no associated CVEs are not included in the exported data (regardless
 
 :::
 
+### Running Images tab
+
+This tab shows CVEs associated with running pods in your cluster so you can assess pod vulnerability and remediate.For example, you can see image exposure by viewing the  **Clusters** and **Running Instances** columns that show the number of running instances in clusters that are connected to {{prodname}}.
+
+### Troubleshooting 
+
+#### No results in the Running Images tab
+
+- The Image Assurance may not be enabled and running
```

Comment:
What does "The Image Assurance" refer to here? 

I think it should say the Runtime view flag.

Reply:
Changed to: The Image Assurance feature may not be enabled.